File: Module 2 - Loops (Iteration)/for loop/for.py
"""
How range() Works (Python)
range() generates a sequence of numbers.

Syntax
range(start, stop, step)
start → First number (default = 0)
stop → Last number is not included
step → Increment (default = 1)
"""

# Example 1 – Print 1 to 5
print("Example 1 -- Print 1 to 5")
for i in range(1, 6):
    print(i)

# Example 2 – Even Numbers
print("Example 2 -- Even Numbers")
for i in range(2, 11, 2):
    print(i)

# Example 3 – Countdown
print("Example 3 -- Countdown")
for i in range(5, 0, -1):
    print(i)

# Example 4 – Multiplication Table
print("Example 4 -- Multiplication Table")
num = 5
for i in range(1, 11):
    print(num, "x", i, "=", num * i)

File: Module 2 - Loops (Iteration)/for loop/for.cpp
/*
for(initialization; condition; update)
{
    statement;
}
*/

#include<iostream>
using namespace std;
#include <iostream>
using namespace std;

int main(){
    
    //Example 1 – Print 1 to 5
    cout << "Example 1 -- Print 1 to 5\n";

    for(int i = 1; i <= 5; i++)
    {
        cout << i << endl;
    }

    cout <<endl;

    // Example 2 – Even Numbers
    cout << "Example -- Even Numbers\n";

    for(int i = 2; i <= 10; i += 2)
    {
        cout << i << endl;
    }
    
    cout <<endl;

    // Example 3 – Countdown
    cout << "Example 3 -- Countdown\n";

    for(int i = 5; i >= 1; i--)
    {
        cout << i << endl;
    }
    
    cout <<endl;

    // Example 4 – Multiplication Table
    cout << "Example 4 -- Multiplication Table\n";int num = 5;

    for(int i = 1; i <= 10; i++)
    {
        cout << num << " x " << i << " = " << num * i << endl;
    }
    
    return 0;
}

